test/integration/roomMembersTest.ts

```diff
@@ -374,4 +377,200 @@ describe("Test: Testing RoomMemberManager", function() {
             }
         }
     });
-});
\ No newline at end of file
+
+    it("!mjolnir since kicks the correct users", async function() {
+        this.timeout(600_000);
```

Comment:
is this 10 minutes? Why's it take so long?

Reply:
Well, we're:

- creating 20 users;
- creating 12 rooms;
- joining each user to each room;
- setting powerlevel in each room;
- kicking/banning half of the users from most rooms.

This regularly exceeds 1 minute on my tests, so I arbitrarily amped the timeout to 10 minutes to have a margin.

Reply:
Okie